direct_db_manager.py

```
@@ -2518,11 +2913,11 @@ def __init__(self, user_info, config_manager):
         self.apply_modern_styles()
 
         # Use single shot timer to avoid blocking
-        QTimer.singleShot(500, self.initial_dashboard_load)
+        QTimer.singleShot(500, self.initial_dashboard_load_async)
```

Ruff (0.12.2)

`QTimer` may be undefined, or defined from star imports

(F405)